One big chunk of code in a feature branch

VicPeralta opened the pull request:
new file:   .eslintrc.json
new file:   .gitignore
new file:   .hintrc
new file:   .stylelintrc.json
new file:   package-lock.json
new file:   package.json
new file:   src/app.js
new file:   src/assets/icons.svg
new file:   src/assets/noun-three-dot-4287657.svg
new file:   src/assets/webpack.svg
new file:   src/index.html
new file:   src/index.js
new file:   src/style.css
new file:   src/tasksList.js
new file:   webpack.config.js

VicPeralta committed:
one big chunk of code in a feature branch

new file:   .eslintrc.json
new file:   .gitignore
new file:   .hintrc
new file:   .stylelintrc.json
new file:   package-lock.json
new file:   package.json
new file:   src/app.js
new file:   src/assets/icons.svg
new file:   src/assets/noun-three-dot-4287657.svg
new file:   src/assets/webpack.svg
new file:   src/index.html
new file:   src/index.js
new file:   src/style.css
new file:   src/tasksList.js
new file:   webpack.config.js

AlejandroNeira98 commented:
Hello Victor,
First of all I want to congratulate you for your amazing work. Your code is very clean and readable 💯  Besides that I love the way that you implemented the functionality of the app entirely through OOP, this is a very efficient and clear way to code 🥇  
Now, as a way to improve your code I have only one recommendation for you: 
- In the following lines I think it would be better if you change the e variable name for a more semantic valuable name. That in order to improve readability.
https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/tasksList.js#L25
https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/tasksList.js#L40
https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/tasksList.js#L69

like you did in this line : https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/app.js#L13

Again, you did an amazing job! Congratulations.

VicPeralta commented:
> Hello Victor, First of all I want to congratulate you for your amazing work. Your code is very clean and readable 💯 Besides that I love the way that you implemented the functionality of the app entirely through OOP, this is a very efficient and clear way to code 🥇 Now, as a way to improve your code I have only one recommendation for you:
> 
> * In the following lines I think it would be better if you change the e variable name for a more semantic valuable name. That in order to improve readability.
>   https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/tasksList.js#L25
>   
>   https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/tasksList.js#L40
>   
>   https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/tasksList.js#L69
> 
> like you did in this line :
> 
> https://github.com/VicPeralta/ToDoReview/blob/7525e6c7eb3880f0ebc66997f3ec0d924bee84a8/src/app.js#L13
> 
> Again, you did an amazing job! Congratulations.

Thank you Alejandro for your review and recommendations.
I have modified the code to take them into account.

VicPeralta committed:
Make changes based on code review

Rename variable e to task in forEach loops and filter method
Create CSS classes to apply style instead of using JS Code

modified:   src/app.js
modified:   src/index.html
modified:   src/style.css
modified:   src/tasksList.js
